### PR TITLE
Close #1050 create a product tab and then connect the product to the taxon

### LINKED ISSUE #1050: Create a product tab and then connect the product to the taxon

### PR DESCRIPTION
**Pull request by Sreyleak-Deth**
Screenshot from 2024-01-31 12-55-15

Screenshot from 2024-02-01 09-11-34

Screenshot from 2024-02-01 09-11-41

Screenshot from 2024-02-01 09-11-50

### CODE REVIEW

File: app/views/spree/admin/classifications/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<% end %>
+
+<% content_for :page_actions do %>
+  <%= button_link_to Spree.t(:add_new_product), new_admin_taxonomy_taxon_classification_path, class: "btn-success", icon: 'add.svg', id: 'admin_new_vendor_link' %>
+<% end if can? :create,  Spree::Classification %>
+
+<%= render partial: 'spree/admin/shared/taxon_tabs', locals: { current: :products } %>
+
+<% if @taxon_products.any? %>
+  <div class="table-responsive border rounded bg-white mb-3">
```
Comment (channainfo) [inline app/views/spree/admin/classifications/index.html.erb:17]: Let's reuse the product partial

**Review by channainfo — CHANGES_REQUESTED** [review]

**Comment by Sreyleak-Deth** [comment]
Noted bong, I will correct

**Comment by Sreyleak-Deth** [comment]
For the product here, it just shows the data of the product that has a relationship with the taxon. And I checked it again, and again, it doesn't have the view of taxon_product; it just has the view of product only. This shows I can't use a partial to render the product list or a product row. What I do here is just show the product_id and product_name that have a connection with the taxon only.

File: app/views/spree/admin/classifications/shared/_products.html.erb [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<%#
+:products
+:empty_message
+%>
+
```
Comment (channainfo) [inline app/views/spree/admin/classifications/shared/_products.html.erb:5]: Put this file in the app/views/spree/admin/shared/_products.html.erb instead.

**Review by channainfo — CHANGES_REQUESTED** [review]

**Pull request merged**